docs/references/observability.rst

```diff
@@ -169,6 +169,20 @@ pgrst_db_pool_max
 
 Max pool connections.
 
+JWT Cache Metric
+----------------
+
+Related to the :ref:`jwt_caching`.
+
+pgrst_jwt_cache_size_bytes
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+======== =======
+**Type** Gauge
+======== =======
+
+Approximate JWT cache size in bytes.
```

Comment:
From the comment on https://github.com/PostgREST/postgrest/pull/3802#issuecomment-2600891340, this is no longer an approximate right? 

```suggestion
The JWT cache size in bytes.
```

Reply:
Yes, this is correct. I forgot to update this.